devices/HumanWrenchProvider/HumanWrenchProvider.cpp

```diff
@@ -483,6 +645,41 @@ bool HumanWrenchProvider::close()
 
 void HumanWrenchProvider::run()
 {
+    if (pImpl->readHumanData) {
+        pImpl->basePositionInterface = pImpl->iHumanState->getBasePosition();
+        pImpl->baseOrientationInterface = pImpl->iHumanState->getBaseOrientation();
+        pImpl->baseVelocityInterface = pImpl->iHumanState->getBaseVelocity();
+        pImpl->jointPositionsInterface = pImpl->iHumanState->getJointPositions();
+        pImpl->jointVelocitiesInterface = pImpl->iHumanState->getJointVelocities();
+        pImpl->jointNamesStateInterface = pImpl->iHumanState->getJointNames();
```

Comment:
with https://github.com/robotology/human-dynamics-estimation/pull/299/commits/fa4b0eebcf0ee11f545579d276de056150861a7a, I added the following changes:
- `basePositionInterface` and `baseVelocityInterface ` are now used to initialize kindyn (it would work even with dummy data for base position and velocity, but I prefer to use them in case in the future we would need to compute other quantities)
- local variables are now used